contracts/WildcardSteward_v2.sol

```diff
@@ -121,6 +130,12 @@ contract WildcardSteward_v2 is Initializable {
         _;
     }
 
+    // This modifier MUST be called anytime before the `benefactorTotalTokenNumerator[benefactor]` value changes.
```

Comment:
Makes lots of sense, the accounting needs to be done every time before the any token price changes of any of the benefactors constituent tokens. 

I'm sure I'll get to it but will make sure it deals with foreclosures.

Reply:
Since, benefactorTotalTokenNumerator[benefactor] should change when a foreclosure happens, guessing this function gets called up until the foreclosure, then benefactorTotalTokenNumerator[benefactor] is adjusted in light of the foreclosure

Reply:
Yes, exactly, we would have a bug if this wasn't called before a foreclosure. But when I checked we had it covered.